src/components/Card/Card.css

```diff
@@ -1,15 +1,11 @@
 .card {
   width: 235px;
-  margin: 0 40px 0 0;
+  margin: 0 0 40px;
```

Comment:
The margin of 40px shown in the brief is too much when you have desktop set in devtools (which adds a vertical scrollbar of about 15px width).  So I removed right margin. 

Also I added bottom margin to improve display when there are more cards than 4.